### PR TITLE
feat(@vtmn/vue): add `VtmnIcon` component

### PR DESCRIPTION
**Pull request by ThomasLaigneauDecathlon**
## Changes description

Add Vue implementation of VtmnIcon component.

## Context

https://github.com/Decathlon/vitamin-web/issues/717

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.
- [x] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/showcases/core/csf/guidelines/iconography.csf.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const argTypes = {
     defaultValue: 24,
     control: {
       type: 'select',
-      options: [16, 24, 32, 64],
+      options: [16, 20, 24, 32, 64],
```
Comment (lauthieb) [inline packages/showcases/core/csf/guidelines/iconography.csf.js:17]: It is allowed in design?

**Review by lauthieb — CHANGES_REQUESTED** [review]
Awesome :tada:
Just one remark

File: packages/showcases/core/csf/guidelines/iconography.csf.js (hunk 1 above)
Comment (ThomasLaigneauDecathlon) [reply packages/showcases/core/csf/guidelines/iconography.csf.js:17]: Added it to match with the React component but it seems that only the 16, 24, and 32 pixels are available. It means we should remove the 20 and 64 size ? We should also make this update in the react component ?

**Review by GaspardMathon — CHANGES_REQUESTED** [review]
See for changing the icon sizes that are not allowed (20, 64)

**Pull request merged**